test/intl402/Temporal/PlainDate/prototype/since/wrapping-at-end-of-month-persian.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2024 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.since
+description: Tests balancing of days to months at end of month (Persian calendar)
+includes: [temporalHelpers.js]
+features: [Temporal, Intl.Era-monthcode]
+---*/
+
+const calendar = "persian";
+
+// Difference between end of longer month to end of following shorter month
+{
+  const end = Temporal.PlainDate.from({ year: 1970, monthCode: "M07", day: 30, calendar });
+  for (const largestUnit of ["years", "months"]) {
+    TemporalHelpers.assertDuration(
+      Temporal.PlainDate.from({ year: 1970, monthCode: "M06", day: 30, calendar }).since(end, { largestUnit }),
+      0, -1, 0, 0, 0, 0, 0, 0, 0, 0,
+      `Shahrivar 30th to Mehr 30th is one month (${largestUnit})`
+    );
+    TemporalHelpers.assertDuration(
+      Temporal.PlainDate.from({ year: 1970, monthCode: "M06", day: 31, calendar }).since(end, { largestUnit }),
+      0, 0, 0, -30, 0, 0, 0, 0, 0, 0,
+      `Shahrivar 31st to Mehr 30th is 30 days, not one month (${largestUnit})`
+    );
+  }
+}
+
+// Difference between end of leap-year Bahman to end of leap-year Esfand
+{
+  const end = Temporal.PlainDate.from({ year: 1973, monthCode: "M12", day: 30, calendar });
+  for (const largestUnit of ["years", "months"]) {
+    TemporalHelpers.assertDuration(
+      Temporal.PlainDate.from({ year: 1973, monthCode: "M11", day: 30, calendar }).since(end, { largestUnit }),
+      0, -1, 0, 0, 0, 0, 0, 0, 0, 0,
+      `Bahman 30th to Esfand 30th is one month (${largestUnit})`
+    );
+  }
+}
+
```

Comment:
So for this one I think it'd better test the behaviour if we used a common year because Esfand has 29 days and is shorter than all the other months, like February in the Gregorian calendar.

Reply:
Fixed in [edce275](https://github.com/tc39/test262/pull/4695/commits/edce2752f57cbb6055235d5ad750c02a00f63917)